app/javascript/src/components/Dashboard/Contacts/utils.jsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+
+import { MenuVertical } from "neetoicons";
+import { Avatar, Typography, Dropdown } from "neetoui";
+
+const renderAvatarWithNameRoleAndImage = (name, { role, imageUrl }) => (
+  <div className="flex flex-row items-center">
+    <Avatar
+      className="mr-2"
+      user={{
+        name,
+        imageUrl,
+      }}
+    />
+    <div>
+      <Typography style="h4">{name}</Typography>
+      <Typography style="body2">{role}</Typography>
+    </div>
+  </div>
+);
+
+const renderDropdownButton = (setShowDeleteAlert, setSelectedContactId, id) => (
+  <div>
+    <Dropdown buttonStyle="text" icon={MenuVertical}>
+      <li> Edit </li>
+      <li
+        onClick={() => {
+          setShowDeleteAlert(true);
+          setSelectedContactId(id);
+        }}
+      >
+        Delete
+      </li>
+    </Dropdown>
+  </div>
+);
+
+export const buildContactColumnData = (
+  setShowDeleteAlert,
+  setSelectedContactId
+) => [
+  {
+    dataIndex: "name",
+    key: "name",
+    title: "NAME & ROLE",
+
+    render: renderAvatarWithNameRoleAndImage,
+  },
+  {
+    dataIndex: "email",
+    key: "email",
+    title: "EMAIL",
+  },
+  {
+    dataIndex: "createdAt",
+    key: "createdAt",
+    title: "CREATED AT",
+
```

Comment:
Not sure why you are adding these blank lines. Remove them. It doesn't help in any manner.